snapcraft/parts/plugins/flutter_plugin.py

```diff
@@ -0,0 +1,121 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""The flutter plugin."""
+from typing import Any, Dict, List, Literal, Set, cast
+
+from craft_parts import infos, plugins
+from overrides import overrides
+
+FLUTTER_REPO = "https://github.com/flutter/flutter.git"
+"""The repository where the flutter SDK resides."""
```

Comment:
should this be a comment instead? I don't know if it makes a difference

Reply:
the constant gets a nice docstring this way (if we ever render it)